README.md

```diff
@@ -117,6 +124,8 @@ require('core-js/build')({
   // ...
 });
 ```
+If your webpack build includes something like UglifyJs, then the excluded parts will be entirely stripped from the resulting code using dead code elimination.
```

Comment:
Sorry, missed this comment earlier. Are you entirely sure about this? I'm pretty certain that if you do

    if(false) { 
        require('something'); 
    }

then webpack just nicely expands the require and leaves it up to a cleanup tool to remove the dead code. (https://webpack.github.io/docs/list-of-hints.html - "Use the DefinePlugin to pass configuration from config to app: Embedded into bundle, Conditional code is removed with minimized").

Or am I just completely misunderstanding how core-js/build works? :-)